lesson04/assignment/__components.scss

```diff
@@ -0,0 +1,94 @@
+// Mixins
+@mixin verticle{
```

Comment:
This will work, but typically we'd want the mixin to take in a value that can be used within the output styles.